### DISCUSSION TITLE
怎么选择声音？

### DISCUSSION THREAD

**Discussion by piwawa** [lobehub/lobehub#4304] [General Question] [answered]
以前可以选不同的声音，现在不能选了，只有一种男声 alloy，官方支持以下声音：

https://platform.openai.com/docs/guides/text-to-speech/voice-options

![image](https://github.com/user-attachments/assets/6d7733c7-3eb2-4dde-a86c-456ac0158362)

**Comment by SpeedupMaster** [comment, marked as answer]
![图片](https://github.com/user-attachments/assets/8d39e50e-c0bb-48ac-b84a-3bc2b1f2c796)
